src/core/data/sagas/deleteEntities.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import {
+  all,
+  call,
+  put,
+  takeEvery,
+} from '@redux-saga/core/effects';
+import { DataApiActions, DataApiSagas } from 'lattice-sagas';
+import { Logger } from 'lattice-utils';
+import type { Saga } from '@redux-saga/core';
+import type { WorkerResponse } from 'lattice-sagas';
+import type { SequenceAction } from 'redux-reqseq';
+
+import { DELETE_ENTITIES, deleteEntities } from '../actions';
+
+const { deleteEntityData } = DataApiActions;
+const { deleteEntityDataWorker } = DataApiSagas;
+
+const LOG = new Logger('DataSagas');
+
+function* deleteEntitiesWorker(action :SequenceAction) :Saga<WorkerResponse> {
+
+  const workerResponse :Object = {};
```

Comment:
Since you're using `WorkerResponse`...

```
  let workerResponse :WorkerResponse;
  try {
    ...
    workerResponse = { data: ??? };
    ...
  }
  catch (error) {
    workerResponse = { error };
    ...
  }
  finally { ... }

  return workerResponse;
```

Reply:
good point! for `workerResponse.data`, I'm returning the number of calls made, because I think the response of `deleteEntityData` is the number of entities deleted.